packages/allure-playwright/src/helpers.ts

```diff
@@ -91,10 +91,12 @@ export class allure {
     });
   }
 
-  static tag(tag: string) {
-    this.label({
-      name: LabelName.TAG,
-      value: tag,
+  static tag(...tag: string[]) {
+    tag.forEach((tags) => {
```

Comment:
```suggestion
    for (let tag of tags)
```
nit: thats usually preferred over .forEach callbacks

Reply:
Corrected, thanks.